README.md

```diff
@@ -1,15 +1,64 @@
 # Flair HQ
 
-## Setup
-
-1. Make sure you have Node (>=4.0.0), and MongoDB installed
-1. Clone the repository `git clone https://github.com/YaManicKill/flairhq.git`
-1. Navigate into the directory `flairhq`
-1. Run `npm install`  to install the dependencies
-1. Copy config/local.example.js to config/local.js
-1. Create a reddit app on [https://www.reddit.com/prefs/apps](https://www.reddit.com/prefs/apps)
-1. Copy the id and secret to config/local.js
-1. Use something like [this](https://github.com/not-an-aardvark/reddit-oauth-helper) to get a refresh token for a moderator on the subs
-1. Start your MongoDB
-1. Start sails with `npm start`
-1. Open `http://localhost:1337` in your browser
+## How to setup your own dev environment
+
+### 1. Create two subreddits for testing
+
+You probably don't to do testing on production so the we suggest to create two subreddits for testing purposes. [Start here](https://www.reddit.com/subreddits/create). You can choose any names for it (such as yourUsername and yourUsername1). Once you have the subreddits created, create another reddit account.  
```

Comment:
"don't want" 

Also, remove "probably".

"we suggest you create 2 subreddits"